File: procon-api/src/main/java/br/com/procon/models/dtos/AtendimentoDto.java
package br.com.procon.models.dtos;

import java.time.LocalDate;
import java.util.List;

/**
 * 
 * @author Marlon F. Garcia
 *
 */
public record AtendimentoDto(Integer id, List<ConsumidorDto> consumidores,
		List<FornecedorDto> fornecedores, LocalDate data, Integer atendente) {

}

File: procon-front/src/app/models/dtos/atendimento-dto.ts
import { ConsumidorDto } from "./consumidor-dto";
import { FornecedorDto } from "./fornecedor-dto";

export interface AtendimentoDto {
  id: number;
  consumidores: ConsumidorDto[];
  fornecedores: FornecedorDto[];
  data: string;
  atendente: number;
}
